src/06-errors_and_files/03-exceptions.md

```diff
@@ -26,7 +26,7 @@ type class. By making a type an instance of the `Exception` type class, we can t
 and catch it in `IO` code:
 
 ```hs
-{-# language LambdaCase #-}
+{-# LANGUAGE LambdaCase #-}
```

Comment:
```suggestion
{-# language LambdaCase #-}
```

Reply:
I don't think I've ever seen it lower case. And the GHC documentation only uses uppercase. Could be worthwhile to point this out if you use lowercase.